mllib/src/main/scala/org/apache/spark/ml/classification/OneVsRest.scala

```diff
@@ -271,7 +273,7 @@ object OneVsRestModel extends MLReadable[OneVsRestModel] {
 @Since("1.4.0")
 final class OneVsRest @Since("1.4.0") (
     @Since("1.4.0") override val uid: String)
-  extends Estimator[OneVsRestModel] with OneVsRestParams with MLWritable {
+  extends Estimator[OneVsRestModel] with OneVsRestParams with HasParallelism with MLWritable {
 
```

Comment:
Maybe we can define
`trait OneVsRestParams extends PredictorParams with HasParallelism`
`trait TrainValidationSplitParams extends ValidatorParams with HasParallelism`
and so on, like other ml algos do ?
But I am not sure whether there is other reasons.